lib/mini_mime.rb

```diff
@@ -146,8 +140,7 @@ def lookup_uncached(val)
       end
 
       def resolve(row)
-        @file.seek(row * @row_length)
-        Info.new(@file.readline)
+        Info.new(@file.pread(@row_length, row * @row_length).force_encoding(Encoding::UTF_8))
```

Comment:
Hum, I really sorry, I realized that some platforms (e.g. Windows) don't have `pread`.

I'll submit another PR to shim it for these.